docs/modules/data-structures/pages/working-with-external-data.adoc

```diff
@@ -170,12 +177,23 @@ class ProcessingStore implements MapStore<Integer, Employee>, PostProcessingMapS
 }
 ----
 
-NOTE: Please note that if you are using post-processing in combination with xref:computing:entry-processor.adoc[Entry Processors], post-processed values will not be carried to in-memory backups.
+NOTE: Please note that if you are editing an object in combination with xref:computing:entry-processor.adoc[Entry Processors], edited values will not be saved to in-memory backups.
+
+== Configuring Hazelcast to use your MapStore Implementation
 
+Once you've created your `MapStore` implementation, you need to configure your map to use it. You have two options:
 
-== Configuring Hazelcast to use your `MapStore` Implementation
+- Use an existing connection.
+- Create a new connection.
 
-Once you've created your `MapStore` implementation, you need to configure your map to use it. 
+[[existing-connection]]
+=== Using an Existing Connection
+
+If you have an xref:external-data-stores:external-data-stores.adoc[external data store] already configured, you can configure your MapStore to use that existing connection.
+
+When you configure your MapStore to use that existing connection, Hazelcast creates a xref:sql:mapping-to-jdbc.adoc[SQL mapping with the JDBC connector]. The name of the mapping is the same name as your map prefixed with `__map-store.`. This mapping is then used to read data from the store, or write data to it. This mapping is removed when the map is removed.
+
+This is a basic example configuration that shows you the minimal configuration that is required. This example is for an external data store configuration called `my-mysql-database`.
```

Comment:
I think this mixes 2 things:
- configuring the MapStore on a map
- the GenericMapStore implementation

They should be separate (either a separate section or even a page). Also, the GenericMapStore should be more prominent (basically at the top). It should be the first option the user should try to use and only if it doesn't work they should resort to own implementation. However it's currently in beta for 5.2, so needs to be marked as such.